### DISCUSSION TITLE
레포지토리 Readme.md (v)

### DISCUSSION THREAD

**Discussion by oceanlee-seoul** [Part3-Team15/taskify#144] [General]
저희 개발 완료하고 저희 프로젝트 잘 요약해서 Readme.md도 업데이트하면 좋을 것 같습니다!!
https://github.com/oceanlee-seoul/frontend
이거 저희 1차 프로젝트 때 Readme인데 다들 어떻게 하셨을까요?!

**Comment by un0211** [comment]
오~ 예쁘게 만드셨네요!! 특히 기술스택!
저희팀은 https://github.com/un0211/ro1ling 이렇게 했어요! Gif로 만들었는데, 시연 영상 찍게되면 그걸로 하는게 더 깔끔할 거 같네요!!

**Reply by oceanlee-seoul** [reply]
우와 상세하게 잘되어있네요!!

**Comment by wjsdncl** [comment]
https://github.com/5Team-Project
https://github.com/wjsdncl/5teamFrontEnd
저희팀은 초반에 하고 나중에 추가를 안했습니다
